mmdet/models/dense_heads/mask2former_head.py

```diff
@@ -125,7 +125,8 @@ def __init__(self,
                         feat_channels, self.decoder_embed_dims, kernel_size=1))
             else:
                 self.decoder_input_projs.append(nn.Identity())
-        self.decoder_positional_encoding = MODELS.build(positional_encoding)
+        self.decoder_positional_encoding = SinePositionalEncoding(
```

Comment:
I think we can keep using positional_encoding rather than positional_encoding_cfg

Reply:
OK, all the related parameter names have been modified.